sqlglot/dialects/bigquery.py

```diff
@@ -165,6 +165,14 @@ class Parser(parser.Parser):
             TokenType.TABLE,
         }
 
+        ID_VAR_TOKENS = {
+            *parser.Parser.ID_VAR_TOKENS,  # type: ignore
+            TokenType.VALUES,
+            TokenType.VIEW,
```

Comment:
why not put this in the base?

Reply:
`VALUES` can't be put in the base parser because in some places we have e.g. `VALUES (1, 2, 3)`, so it tries to parse it as aliases and fails. We can put `VIEW` in there, though. Do you think we should make this change?

Reply:
can other system uses view as an alia’s?

Reply:
Seems like postgres and duckdb allow it. Making the change in a sec.